web/pandas/config.yml

```diff
@@ -99,30 +99,50 @@ maintainers:
   - Wes McKinney
   - Jeff Reback
   - Joris Van den Bossche
-partners:
+sponsors:
   active:
   - name: "NumFOCUS"
     url: https://numfocus.org/
     logo: /static/img/partners/numfocus.svg
+    kind: numfocus
   - name: "Anaconda"
     url: https://www.anaconda.com/
     logo: /static/img/partners/anaconda.svg
-    employs: "Tom Augspurger, Brock Mendel"
+    kind: partner
+    description: "Tom Augspurger, Brock Mendel"
   - name: "Two Sigma"
     url: https://www.twosigma.com/
     logo: /static/img/partners/two_sigma.svg
-    employs: "Phillip Cloud, Jeff Reback"
+    kind: partner
+    description: "Phillip Cloud, Jeff Reback"
   - name: "RStudio"
     url: https://www.rstudio.com/
     logo: /static/img/partners/r_studio.svg
-    employs: "Wes McKinney"
+    kind: partner
+    description: "Wes McKinney"
   - name: "Ursa Labs"
     url: https://ursalabs.org/
     logo: /static/img/partners/ursa_labs.svg
-    employs: "Wes McKinney, Joris Van den Bossche"
+    kind: partner
+    description: "Wes McKinney, Joris Van den Bossche"
   - name: "Tidelift"
     url: https://tidelift.com
     logo: /static/img/partners/tidelift.svg
+    kind: regular
+    description: "<i>pandas</i> is part of the <a href=\"https://tidelift.com/subscription/pkg/pypi-pandas?utm_source=pypi-pandas&utm_medium=referral&utm_campaign=readme\">Tidelift subscription</a>. You can support pandas by becoming a Tidelift subscriber."
+  - name: "Chan Zuckerberg Initiative"
+    url: https://chanzuckerberg.com/
+    logo: /static/img/partners/czi.svg
```

Comment:
Can this be pulled from the web rather than included in our repo?

Reply:
That's surely an option. My preference is to copy the image, since it's quite small, and we have control and certainty about its content (it won't break if CZI renames the file, or we won't get a logo with a santa hat for xmas, if they decide this looks cool in their website...).

Reply:
I'd rather pull from the web; one less file in the repo and on the flip side if they update we aren't out of date

Reply:
If we do that, we should do it for all the sponsors that have the logo we need online. Doing it just for CZI doesn't make a lot of sense.

I can remove CZI from this PR, so we can get the rest of the styling and cleaning merged, and we can work on the logos and add CZI later.

Reply:
Oh didn't realize we did this for others. OK for now then